EventFilter/L1TRawToDigi/plugins/implementations_stage2/GMTSetup.cc

```diff
@@ -103,6 +106,9 @@ namespace l1t {
       // input muons on links 36-71
       auto gmt_in_unp = static_pointer_cast<l1t::stage2::RegionalMuonGMTUnpacker>(
           UnpackerFactory::get()->make("stage2::RegionalMuonGMTUnpacker"));
+      if (fw >= 0x8010000) {
+        gmt_in_unp->setUseEmtfDisplacementInfo();
```

Comment:
Out of curiosity: why is this `Emtf` and not `Omtf` ?

Reply:
Hi @missirol,

you're absolutely right, this is a bug. Apologies. Will commit the fix either today or tomorrow.

Cheers,
Dinyar

Reply:
Committed a fix

Reply:
Thanks !